go.mod

```diff
@@ -3,6 +3,7 @@ module gopkg.in/DataDog/dd-trace-go.v1
 go 1.12
 
 require (
+	github.com/DataDog/datadog-agent/pkg/obfuscate v0.0.0-20211129110424-6491aa3bf583
```

Comment:
Any chance we can get a proper mod version here?

Reply:
Not yet. We'll have to use the pseudo-version format until the Datadog Agent is released, and an official tag is assigned to this module. It should be fine. Do you see any issues with this approach, until that happens?